openapi3.yaml

```diff
@@ -49,17 +42,326 @@ components:
       properties:
         message:
           type: string
-    resource:
-      required:
-        - id
-        - name
-        - description
+    convert:
       type: object
+      required:
+        - type
+        - properties
+        - geometry
       properties:
-        id:
-          type: number
-          format: int64
-        name:
-          type: string
-        description:
+        type:
           type: string
+          enum:
+            - Feature
+        properties:
+          type: object
```

Comment:
please add an example of a free form object with some properties and values

Reply:
Done